packages/ui-kit/src/components/AccessTokenProvider/index.ts

```diff
@@ -0,0 +1 @@
+export * from './AccessTokenProvider'
```

Comment:
Let's be selective with the exports. Remember: these form part of our public API. So what do we need to export?

- AccessTokenProvider
- AccessTokenProviderProps
- useAccessToken
- UseAccessToken

Reply:
Right, in this case only `AccessTokenProvider` and `AccessTokenProviderProps` should be exposed